src/NetworkTest/testConnectivity/index.ts

```diff
@@ -252,9 +262,10 @@ function checkSubscribeToSession({ session, publisher }: PublishToSessionResults
 /**
  * Attempt to connect to the tokbox client logging server
  */
-function checkLoggingServer(OT: OT.Client, input?: SubscribeToSessionResults): Promise<SubscribeToSessionResults> {
+function checkLoggingServer(OT: OT.Client, options?: NetworkTestOptions, input?: SubscribeToSessionResults): Promise<SubscribeToSessionResults> {
   return new Promise((resolve, reject) => {
-    const url = `${getOr('', 'properties.loggingURL', OT)}/logging/ClientEvent`;
+    const loggingUrl = 'hlg.tokbox.com/prod/logging/ClientEvent';
```

Comment:
I removed the `properties.loggingUrl` because when I use it, the Library logs a warning in the console saying `properties.loggingUrl will be deprecated in the next version`

Reply:
We may use different logging URLs in the standard and enterprise environments. Let me check....

Reply:
Yes, the enterprise line does use a different logging URL. Please use `OT. properties.loggingUrl`. We are aware that it logs a warning to the console (and we are leaving that as a known issue, for now).